tests/DucksNet.IntegrationTests/MedicalRecordsControllerTests.cs

```diff
@@ -56,9 +58,11 @@ public async Task When_CreatedMedicalRecordWithEmptyIdClient_Then_ShouldReturnFa
 
         MedicalRecordDTO medicalRecordDTO = new MedicalRecordDTO(idAppointment, idClient);
         //Act
-        var medicalRecordResponse = await TestingClient.PostAsJsonAsync(ApiURL, medicalRecordDTO);
-        var medicalRecordResult = await TestingClient.GetAsync(ApiURL);
+        var medicalRecordResponse = await TestingClient.PostAsJsonAsync(MedicalRecordURL, medicalRecordDTO);
+        var medicalRecordResult = await TestingClient.GetAsync(MedicalRecordURL);
         //Assert
         medicalRecordResponse.StatusCode.Should().Be(HttpStatusCode.BadRequest);
     }
+
+    // TODO(RO) : after push of pet, can start here
```

Comment:
why do your tests need to be ordered? You can call CleanDatabase() at the start of each test to ensure a clean enviroment.

Reply:
I need the new modifications for the Pet Domain. In this branch, Pet has only the size member